libpod/pod.go

```diff
@@ -128,6 +138,17 @@ func (p *Pod) Namespace() string {
 	return p.config.Namespace
 }
 
+// ResourceLim returns the cpuset resource limits for the pod
+func (p *Pod) ResourceLim() *spec.LinuxResources {
+	resCopy := p.config.ResourceLimits
+	return resCopy
+}
+
+// CPUs returns the specified number of CPUs for the pod
+func (p *Pod) CPUs() float64 {
```

Comment:
```suggestion
func (p *Pod) CPUs() int {
```
NumCPU() signature is int, not float64